### DISCUSSION TITLE
Are there any training receipts for the canary model?

### DISCUSSION THREAD

**Discussion by literate-goggles** [NVIDIA-NeMo/NeMo#9437] [Q&A]
Hi, I would like to experiment with the encoder-decoder canary model, but I couldn't find any default configurations or training scripts for it in the main branch

**Comment by titu1994** [comment]
It's usher speech_multitask - https://github.com/NVIDIA/NeMo/blob/main/examples/asr/conf/speech_multitask/fast-conformer_aed.yaml

We'll add better documentation for the 2.0 release. @krishnacpuvvada fyi
